src/layouts/MenuLayout/MenuLayout.js

```diff
@@ -7,9 +7,8 @@ import MenuPanel from 'components/MenuPanel'
 import OverlayNotification from 'components/OverlayNotification'
 
 type Props = {|
-  +openLayout: () => void,
-  +closeLayout: () => void,
-  +onSendAssetClick: () => void,
+  +openLayout: Function,
+  +closeLayout: Function,
```

Comment:
why did you remove types for functions' signature?

Reply:
Because any signature would do the trick

Reply:
sorry, don't understand.
You just removed valid types for function signature.

`() => void` means that function doesn't get any params and return nothing. And it is true for specific redux actions

Reply:
Yeah, but React passes arguments to functions and works with their output, so we just lie to ourselves without reason

Reply:
But personally I don't care for those types because we don't use them as intended.

Reverted

Reply:
> Yeah, but React passes arguments to functions and works with their output

where did you get such info?

> so we just lie to ourselves without reason

we use and call our functions as per we said about it to `flow`. How we lie to ourselves?